src/sagemaker/jumpstart/factory/model.py

```diff
@@ -565,6 +569,14 @@ def _add_config_name_to_kwargs(kwargs: JumpStartModelInitKwargs) -> JumpStartMod
             kwargs.config_name or specs.inference_configs.get_top_config_from_ranking().config_name
         )
 
+        if kwargs.config_name:
```

Comment:
Are we initializing config_name as None? if not we should use getattr/hasattr to avoid attribute not found exception

Reply:
We are initializing config_name as None similar to other fields across the kwargs class, this should be fine